src/mlpack/core/optimizers/minibatch_sgd/decay_policies/no_decay.hpp

```diff
@@ -32,13 +32,12 @@ class NoDecay
   /**
    * This function is called in each iteration after the policy update.
    *
-  * @param iterate Parameters that minimize the function.
-  * @param stepSize Step size to be used for the given iteration.
-  * @param gradient The gradient matrix.
-  */
-  void Update(arma::mat& /* iterate */,
-              double& /* stepSize */,
-              const arma::mat& /* gradient */)
+   * @param stepSize Step size to be used for the given iteration.
+   * @param epoch Size of one epoch
+   * @param decaySteps decay learning after these many steps
+   */
+  void Update(double& /*stepSize*/,
```

Comment:
The same comment here. I think these changes need a separate discussion. That's why I removed that commit and did a branch from the previous one.